canonical-web-and-design/maas-ui
# Add double row component

## Done
- Replace double row elements with a component.
- Use the double row component to display a contextual menu.

## QA
- Visit /r/machines.
- Hover rows, the menu icon should appear next to the owner.
- Clicking the menu icon should display the menu and keep the row highlighted.

## Fixes
Fixes https://github.com/canonical-web-and-design/MAAS-squad/issues/1797.
